### ISSUE TITLE
aur sync "${packages[@]}" --no-ver-shallow returns ==> ERROR: build: invalid argument (no packages found)

### ISSUE THREAD

**Issue by CarbonChauvinist** [AladW/aurutils#740]
[aurutils.log](https://github.com/AladW/aurutils/files/5249849/aurutils.log)
`~ $ pacman -Qs aurutils                                                                                                                                                                                                     
local/aurutils 2.3.5-1                                                                                                                                                                                               
    helper tools for the arch user repository `

I use some very simple functions to assist in handling vcs packages per the recommended steps from the man pages. 

```
~ $ type vcs-sync
vcs-sync is a function
vcs-sync () 
{ 
    mapfile -t mypacks < <(aur vercmp-devel | cut -d: -f1);
    if [ "${#mypacks[@]}" = 0 ]; then
        printf "no vcs updates";
    else
        echo "following ${#mypacks[@]} package(s) have update(s):" "${mypacks[@]}";
    fi
}

~ $ type vcs-up
vcs-up is a function
vcs-up () 
{ 
    if (( ${#mypacks[@]} )); then
        aur sync "${mypacks[@]}" --no-ver-shallow;
        unset mypacks;
    else
        printf "nah brah";
    fi
}
```
Since updating to aurutils 2.3.5-1 attempting to sync these packages with the `--no-ver-shallow` flag returns an error that seems as if no packages are actually being passed to aur build? (AFAICU anyway)

```
~ $ echo "${mypacks[@]}"
git-delta-git libucl-git wayst-git

~ $ AUR_DEBUG=1 aur sync "${mypacks[@]}" --no-ver-shallow 2>&1 | tee -a aurutils.log                       
==> Using [aur] repository
    git-delta-git: r.1133.759f484-1 is newer than r.319.f89300a-1
    libucl-git: 0.8.1.r128.ga931a15-1 is newer than 0.7.3.r113.g20b1288-1
    wayst-git: r156.1a46142-1 is newer than r146.18a038b-1
==> Retrieving package files
From https://aur.archlinux.org/wayst-git
 = [up to date]      master     -> origin/master
From https://aur.archlinux.org/libucl-git
 = [up to date]      master     -> origin/master
From https://aur.archlinux.org/git-delta-git
 = [up to date]      master     -> origin/master
Running makepkg --clean --syncdeps
==> ERROR: build: invalid argument (no packages found)
```

Not sure how useful the debug log will be, but it's attached.  The same thing happens when bypassing my functions and running the commands directly; Is there anything I should be doing differently?

**Comment by AladW** [comment]
Please try 6365e14fe694caa01e0e73641826969743f4e307 in 2.3 branch

**Comment by prietus** [comment]
ef1e79d5e892dda9da3dd6801294311d0e041124 in 2.3 fixes the issue
thanks

**Comment by AladW** [comment]
Thanks, released 2.3.6 👍

**Comment by CarbonChauvinist** [comment]
Thanks!
